### PR TITLE
Documentation about automatic update checks.

### PR DESCRIPTION
**Pull request by simonwo**

### CODE REVIEW

File: docs/next-steps/update-checks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+If an update is available, an INFO level message will be printed to the log.
+
+## Configuring checks
+
+Bacalhau has some configuration options for controlling how often checks are performed:
+
+| Config property | Environment variable | Default value | Meaning |
+|---|---|---|---|
+| Update.SkipChecks | `BACALHAU_UPDATE_SKIPCHECKS` | False | If true, no update checks will be performed. |
```
Comment (rossjones) [inline docs/next-steps/update-checks.md:32]: What does 'If true' mean here?  The value 'true'? Or does the presence of any value mean true?

File: docs/next-steps/update-checks.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+| Update.CheckFrequency | `BACALHAU_UPDATE_CHECKFREQUENCY` | 24 hours | The minimum amount of time between automated update checks. |
```
Comment (rossjones) [inline docs/next-steps/update-checks.md:33]: If the default is "24 hours" can I specify "8 hours"?  Do I use 24h, 15m etc?

**Review by rossjones — APPROVED** [review]
Looks good, minor suggestions around the env vars.

File: docs/next-steps/update-checks.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+| Update.CheckStatePath | `BACALHAU_UPDATE_CHECKSTATEPATH` | $BACALHAU_DIR/update.json | Where Bacalhau should store the date and time of the last check. |
```
[the remark this answers is not in the document]
Comment (simonwo) [reply docs/next-steps/update-checks.md:34]: It has actually proved useful! https://bacalhauproject.slack.com/archives/C052BAZJVEU/p1697633400854479?thread_ts=1697058908.618639&cid=C052BAZJVEU

**Review by js-ts — APPROVED** [review]

**Pull request merged**